Readme.md

```diff
@@ -87,19 +85,18 @@ Hostname: ubuntu-impish
 
 In some cases, you might want to leverage Tracee's eBPF event collection
 capabilities directly, without involving the detection engine. This might be
-useful for debugging/troubleshooting/analysis/research/education.
+useful for debugging, troubleshooting, analysis, research, or educational purposes.
 
-Just execute docker with `trace` argument first, and tracee-ebpf will be
+Just execute docker with the `TRACEE_EBPF_ONLY=1` environment variable, and Tracee-eBPF will be
 executed, instead of the full tracee detection engine.
 
 ```shell
-docker run \
-  --name tracee --rm -it \
+docker run --name tracee --rm -it \
   --pid=host --cgroupns=host --privileged \
   -v /etc/os-release:/etc/os-release-host:ro \
   -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
-  aquasec/tracee:latest \
-  trace
+  -e TRACEE_EBPF_ONLY=1 \
```

Comment:
I've added this temporarily, until we bring back the trace subcommand.

Reply:
@rafaeldtinoco I also see that in your PR you removed these lines:

> ## Documentation
> 
> The full documentation of Tracee is available at [https://aquasecurity.github.io/tracee/latest](https://aquasecurity.github.io/tracee/latest). You can use the version selector on top to view documentation for a specific version of Tracee.

I think that we need to get them back

Reply:
Thanks for heads up. It's hard to follow what is going on so I just fixed image ref in scope of this PR